SwanCustomEnvironments/swancustomenvironments/serverextension.py

```diff
@@ -31,37 +33,17 @@ async def get(self):
         self.set_header("Content-Type", "text/event-stream")
         makenv_process = SwanCustomEnvironmentsApiHandler.makenv_process
         
-        log_file = None
         if makenv_process is None:
-            # The first get request will launch the execution of the makenv.sh script,
-            # to create the environment
             makenv_process = self._launch_makenv()
-            SwanCustomEnvironmentsApiHandler.makenv_process = makenv_process
```

Comment:
You can just do this assignment above, instead of doing it in the helper function? The rational is that _process_first_get is just in charge of reading output and sending to frontend, just like process_subsequent_get.